### PR TITLE
make directory if necessary

### PR DESCRIPTION
**Pull request by ohayon**
Fixes https://github.com/fastlane/pem/issues/77

### CODE REVIEW

File: lib/pem/manager.rb [hunk 1]
```diff
@@ -68,7 +68,9 @@ def create_certificate
         end
 
         if PEM.config[:generate_p12]
-          p12_cert_path = File.join(PEM.config[:output_path], "#{filename_base}.p12")
+          output_path = PEM.config[:output_path]
+          FileUtils.mkdir_p(File.expand_path(output_path)) unless Dir.exist?(File.expand_path(output_path))
```
Comment (KrauseFx) [inline lib/pem/manager.rb]: You can remote the `Dir.exist?(File.expand_path(output_path))` as `mkdir_p` will not fail if the directory is already there

**Comment by KrauseFx** [comment]
Looks great, I added one comment before it's ready :+1:

**Comment by ohwutup** [comment]
thanks @KrauseFx! just fixed it up, let me know if thats cool! :octopus:

**Comment by KrauseFx** [comment]
Yeah, thanks @ohwutup :rocket:

**Pull request merged**